model/gen-ai/spans.yaml

```diff
@@ -276,6 +277,15 @@ groups:
         requirement_level: opt_in
       - ref: gen_ai.output.messages
         requirement_level: opt_in
+      - ref: gen_ai.tool.definitions
+        requirement_level: opt_in      
+      - ref: gen_ai.orchestrator.agent_definitions
+        requirement_level:
+          conditionally_required: if applicable 
+        note: |
+          Information about child agents utilized during execution.
```

Comment:
This note overrides the original longer note, making that one less likely to be seen. Is that intentional?